packages/react-lowcode/src/codegen/tests/list/facade.test.ts

```diff
@@ -18,13 +18,13 @@ describe("codegen facade", () => {
  
     test(".add column to existing table", () => {
       let facade = new GeneratorFacade();
-      facade.insertColumn({lineNumber: 15,columnNumber: 73, fileName: 'D:\\Dev\\LowCode\\lowcode-fork\\lowcode\\packages\\react-lowcode\\src\\codegen\\tests\\list\\facade-test-file.ts'}, {entityField: getTestProperty('name')[0]})
+      facade.insertColumn({lineNumber: 15,columnNumber: 73, fileName: 'D:\\Dev\\LowCode\\lowcode-fork\\lowcode\\packages\\react-lowcode\\src\\codegen\\tests\\list\\list-test-file.txt'}, {entityField: getTestProperty('name')[0]})
 
     });
 
     test(".add widget to existing detail page", () => {
       let facade = new GeneratorFacade();
-      facade.insertFormWidget({lineNumber: 37,columnNumber: 0, fileName: 'C:\\Private\\mat-app\\lowcode\\packages\\react-lowcode\\src\\codegen\\tests\\list\\datail-test-file-ast.txt'}, {entityField: getTestProperty('name')[0]})
+      facade.insertFormWidget({lineNumber: 38,columnNumber: 19, fileName: 'D:\\Dev\\LowCode\\lowcode-fork\\lowcode\\packages\\react-lowcode\\src\\codegen\\tests\\list\\datail-test-file.txt'}, {entityField: getTestProperty('name')[0]})
```

Comment:
Please do not hardcode absolute path in the test and use:
- use test data for the in this repo
- standard `snapshot` mechanism for asserting

For end-to-end test you can make codegen CLI as devdependency directly in th example-material-ui (I coud deploy also preview of pull requests in that project in netlify/vecel).